src/utilities/params.test.ts

```diff
@@ -61,13 +61,13 @@ describe('setParamValue', () => {
   test('returns for correct value for Boolean', () => {
     expect(setParamValue(true, Boolean)).toBe('true')
     expect(setParamValue(false, Boolean)).toBe('false')
-    expect(() => setParamValue('', Boolean)).toThrowError()
+    expect(() => setParamValue('' as unknown as boolean, Boolean)).toThrowError()
```

Comment:
I would maybe use a `@ts-expect-error` rather than `as` here that way if it every doesn't error ts will tell us rather than always just casting to the correct value.